app/templates/components/forms/orders/order-form.hbs

```diff
@@ -230,7 +230,25 @@
                     @label={{t "I agree that the event organizer is allowed to take photo, video and/or audio recordings of me or my profile and also texts created by me in the chats which might be published on the event organizer's websites and/or social media channels."}}
                     @checked={{holder.acceptVideoRecording}}         
                     @name={{if field.isRequired (concat field.fieldIdentifier "_required_" index) (concat field.fieldIdentifier "_" index)}}       
-                    @onChange={{action (mut holder.acceptVideoRecording)}}/> 
+                    @onChange={{action (mut holder.acceptVideoRecording)}}/>
+                {{else if (eq field.fieldIdentifier 'is_consent_form_field')}}
+                  <UiCheckbox
+                    @checked={{get holder field.identifierPath}}         
+                    @name={{if field.isRequired (concat field.fieldIdentifier "_required_" index) (concat field.fieldIdentifier "_" index)}}       
+                    @onChange={{action (mut (get holder field.identifierPath))}} />
+                    <span>{{t 'I agree to the terms of the'}} </span>
+                    <a href="https://foundation.wikimedia.org/wiki/Policy:Universal_Code_of_Conduct">{{t 'Universal Code of Conduct'}}</a>
+                    <span> {{t 'and the'}} </span>
+                    <a href="https://foundation.wikimedia.org/wiki/Policy:Friendly_space_policy">{{t 'Friendly Space Policy'}}</a>
+                    <span> .</span>
+                {{else if (eq field.type 'checkbox')}}
+                  {{#if (and (not-eq field.fieldIdentifier "language_form_1") (not-eq field.fieldIdentifier "language_form_2"))}}     
```

Comment:
is there a better naming convention other than language_form_1?

Reply:
Hi @cweitat.

Sorry for this name. I implemented this name on ticket [#8788](https://github.com/fossasia/open-event-frontend/issues/8788). So I will re-open this ticket and convert new name

Thanks